### DISCUSSION TITLE
[FEATURE] As we are using free service of AdminLte , my question is will it provide us future support?

### DISCUSSION THREAD

**Discussion by taherbth** [ColorlibHQ/AdminLTE#3339] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by REJack** [comment, marked as answer]
AdminLTE is only a free service 🤣 we don't offer licenses to buy. You will recieve support till the version is still maintained.
